src/persist-client/src/batch.rs

```diff
@@ -352,7 +352,7 @@ pub(crate) const BATCH_DELETE_ENABLED: Config<bool> = Config::new(
 pub(crate) const BATCH_COLUMNAR_FORMAT: Config<&'static str> = Config::new(
     "persist_batch_columnar_format",
     BatchColumnarFormat::default().as_str(),
-    "Columnar format for a batch written to Persist, either 'row', 'both', or 'both_v1' (Materialize).",
+    "Columnar format for a batch written to Persist, either 'row' or 'both_v2' (Materialize).",
```

Comment:
If we're removing valid values, are we sure those don't exist in the wild anywhere?

Reply:
We actually didn't remove `'both'`, I updated the comment. Good call out, I'm positive `both_v1` is never used in the wild